### PR TITLE
bug(): Adding es6 shims and system polyfills. Those are required by IE.

### PR DESCRIPTION
**Pull request by hansl**

### CODE REVIEW

File: lib/broccoli/angular2-app.js [hunk 1]
```diff
@@ -41,8 +41,10 @@ Angular2App.prototype.toTree = function () {
         'angular2/bundles/http.dev.js',
         'angular2/bundles/router.dev.js',
         'angular2/bundles/upgrade.dev.js',
+        'es6-shim/es6-shim.js'
         'rxjs/bundles/Rx.js',
         'systemjs/dist/system.src.js'
```
Comment (filipesilva) [inline lib/broccoli/angular2-app.js]: I think you forgot a comma here.

File: lib/broccoli/angular2-app.js [hunk 2]
```diff
@@ -41,8 +41,10 @@ Angular2App.prototype.toTree = function () {
         'angular2/bundles/http.dev.js',
         'angular2/bundles/router.dev.js',
         'angular2/bundles/upgrade.dev.js',
+        'es6-shim/es6-shim.js',
         'rxjs/bundles/Rx.js',
         'systemjs/dist/system.src.js'
```
Comment (IgorMinar) [inline lib/broccoli/angular2-app.js]: comma missing

**Comment by IgorMinar** [comment]
can you also add a conditional comment to to the index.html blueprint to load these scripts on older IEs

**Comment by IgorMinar** [comment]
otherwise lgtm

**Comment by filipesilva** [comment]
@hansl could you rebase on master? Then we can merge it.

**Pull request merged**